doc/pages/source/documentation/index.md

```diff
@@ -950,7 +950,24 @@ to the chain of middleware execution.
 ### Sub-plans
 
 -   *when to use?*
--   *how to use?*
+
+To use sub-plans, you must include the `Dynflow::Action::WithSubPlans` module
+and override the `create_sub_plans` method. Inside the `create_sub_plans`
+method, you use the `trigger` method to create sub-tasks that will be executed
+in sequence but will not block the parent task.
```

Comment:
> you use the `trigger` method to create sub-tasks that will be executed in sequence

That's not entirely correct. The spawned sub-tasks are usually planned in sequence, but there are no other constraints on how they are actually executed (the run and finalize phase) and because of that we can't really say they're "executed in sequence".